partials/social-links-button.hbs

```diff
@@ -0,0 +1,96 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+
+
+<style>
```

Comment:
I would suggest placing the classes in a separate css file and import them in here